powerline/segments/i3wm.py

```diff
@@ -1,9 +1,9 @@
 # vim:fileencoding=utf-8:noet
 from __future__ import (unicode_literals, division, absolute_import, print_function)
 
-import i3
-
+import i3ipc
 
+conn = i3ipc.Connection()
```

Comment:
But then it would create a new connection every time. Should I instead use `ThreadedSegment`?